### ISSUE TITLE
GroupItem.members (really HABApp.openhab.item_to_reg.get_members ) fails due to invalid type comparisons

### ISSUE THREAD

**Issue by alasla** [spacemanspiff2007/HABApp#289]
https://github.com/spacemanspiff2007/HABApp/blob/a123fbfa9928ebb3cda9a84f6984dcba593c8236/src/HABApp/openhab/item_to_reg.py#L67

```
def get_members(group_name: str) -> Tuple['HABApp.openhab.items.OpenhabItem', ...]:
    ret = []
    for name in MEMBERS.get(group_name, []):
        ret.append(Items.get_item(name))
    return tuple(sorted(ret))
```

Its not really safe to sort these item objects.  I get lots of errors about invalid comparisons during the sort.  Examples:

```
022-06-11 23:48:33.689 [ERROR] [HABApp.Rules                        ] -      self = <ContactItem name: SensorDenWindow, value: None, last_change: 2022-06-11T22:55:53.756869, last_update: 2022-06-11T22:55:53.756869>
2022-06-11 23:48:33.689 [ERROR] [HABApp.Rules                        ] -      other = <SwitchItem name: SwitchGarageExt, value: OFF, last_change: 2022-06-11T22:55:53.757106, last_update: 2022-06-11T23:35:58.160545>
...
2022-06-11 23:48:33.694 [ERROR] [HABApp.Rules                        ] - TypeError: '>' not supported between instances of 'str' and 'NoneType'
```

This comes up when doing GroupItem.members which uses the get_members() function and fails if any comparisons happen between str and NoneType.  

Not sure how crucial the sorting is, but it either needs to not be sorted, or the sorting func needs to be able to handle different types.

**Comment by spacemanspiff2007** [comment]
Hi - thank you for reporting this issue.

I've fixed this issue already in the dev branch - but the dev branch has other breaking changes so I've not published it yet.
I guess in (about) the next four to six weeks there will be a release which consolidates all the fixes.

**Comment by spacemanspiff2007** [comment]
Fixed with 1.0
